src/cryptoadvance/specter/server_endpoints/wallets.py

```diff
@@ -1000,3 +961,178 @@ def rescan_progress(wallet_alias):
     except SpecterError as se:
         app.logger.error("SpecterError while get wallet rescan_progress: %s" % se)
         return {}
+
+
+@wallets_endpoint.route("/wallet/<wallet_alias>/get_label", methods=["POST"])
+@login_required
+def get_label(wallet_alias):
+    try:
+        wallet = app.specter.wallet_manager.get_by_alias(wallet_alias)
+        address = request.form.get("address", "")
+        label = wallet.getlabel(address)
+        return {
+            "address": address,
+            "label": label,
+        }
+    except Exception as e:
+        flash("Error while get label: %s" % e, "error")
```

Comment:
Flashing to the user doesn't make sense in API-calls. What about:
```suggestion
        logging.exception(e)
        return "Error while get_label: %s" % e, 500
```
Imho that's valid in other cases as well. Will reference it as "see above".